### ISSUE TITLE
ERR | SQL-ERR 'SQLSTATE[42000]: Syntax error

### ISSUE THREAD

**Issue by shaun-ba** [connortechnology/ZoneMinder#81]
Another ERR i came across in logs, thought i'd log it.

```
2018-01-24 13:36:04.324562 | web_php |   | 238 | ERR | SQL-ERR 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '-25, 25' at line 1', statement was 'SELECT E.*,M.Name AS MonitorName,M.DefaultScale FROM Monitors AS M INNER JOIN Events AS E on (M.Id = E.MonitorId) WHERE 1 and ( E.StartTime >= '2018-01-24 12:36:04' and E.MonitorId = '2' ) ORDER BY E.StartTime asc limit -25, 25' params:
-- | -- | -- | -- | -- | --
```

**Comment by connortechnology** [comment]
The limit is a text box, you have entered -25,25 into it.  I will have to add filtering of non-numeric input.

**Comment by shaun-ba** [comment]
What box? No box in the filters anywhere that i can see has -25,25 in it

**Comment by shaun-ba** [comment]
fixed in 1.31.29
